recipe/patches/ignore-hdf5-bounds-check-cmake.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/a/config/cmake_files/medMacros.cmake b/b/config/cmake_files/medMacros.cmake
+index 79698ad..ca4350b 100644
+--- a/a/config/cmake_files/medMacros.cmake
++++ b/b/config/cmake_files/medMacros.cmake
+@@ -444,16 +444,11 @@ MACRO(MED_FIND_HDF5)
+     
+     FIND_PACKAGE(MedfileHDF5 REQUIRED)
+ 
+-    ##
+-    ## Requires 1.10.x version
+-    ##
+-    IF (NOT HDF_VERSION_MAJOR_REF EQUAL 1 OR NOT HDF_VERSION_MINOR_REF EQUAL 10 OR NOT HDF_VERSION_RELEASE_REF GREATER 1)
+-        MESSAGE(FATAL_ERROR "HDF5 version is ${HDF_VERSION_REF}. Only versions >= 1.10.2 are supported.")
+-    ENDIF()
+-    ##
+-    ##
+-
+     ADD_DEFINITIONS(-DH5_USE_16_API)  
```

Comment:
This is really weird. to my knowledge, looking at the HDF5 documentation, it seems that 1.6 didn't define `H5O`

https://docs.hdfgroup.org/archive/support/HDF5/doc1.6/RM_H5Front.html


https://docs.hdfgroup.org/archive/support/HDF5/doc1.8/RM/RM_H5O.html

Reply:
They are basically using a hybrid 1.6 and 1.10 API, so the macro symbols that aren't define in 1.6 just get their default from their 1.10 version.

Bad bad bad.

Reply:
Please see https://github.com/HDFGroup/hdf5/issues/3797 

The compatibility level in your files is set by `H5Pset_libver_bounds`